src/explore-education-statistics-frontend/src/modules/release-publishing-feedback/ReleasePublishingFeedbackPage.tsx

```diff
@@ -0,0 +1,154 @@
+    <Page
+      title="Your recent publishing experience"
+      caption="Feedback"
+      breadcrumbLabel="Feedback"
+      breadcrumbs={[
+        { name: 'Find statistics and data', link: '/find-statistics' },
+      ]}
+    >
+      <Head>
+        <meta name="robots" content="noindex,nofollow" />
+        <meta name="googlebot" content="noindex,nofollow" />
+      </Head>
+      <FormProvider
+        enableReinitialize
+        initialValues={{
+          response: initialResponse,
+        }}
+        validationSchema={Yup.object({
+          response: Yup.string()
+            .required('Select a response')
+            .oneOf<ReleasePublishingFeedbackResponse>([
+              'ExtremelySatisfied',
+              'VerySatisfied',
+              'Satisfied',
+              'SlightlyDissatisfied',
+              'NotSatisfiedAtAll',
+            ]),
+          additionalFeedback: Yup.string().max(
+            2000,
+            'Additional feedback must be 2000 characters or less',
+          ),
+        })}
+      >
+        {({ formState }) => (
+          <LoadingSpinner
+            loading={formState.isSubmitting || formState.isLoading}
+            alert
+            text="Submitting feedback"
+          >
+            {!formState.isSubmitSuccessful ? (
+              <Form
+                id="releasePublishingFeedbackForm"
+                onSubmit={handleFormSubmit}
+              >
+                <FormFieldRadioGroup<FormValues>
+                  legend="How satisfied were you with your recent publishing experience?"
+                  name="response"
+                  order={[]}
+                  options={[
+                    {
+                      label: 'Extremely satisfied',
+                      value: 'ExtremelySatisfied',
+                    },
+                    {
+                      label: 'Very satisfied',
+                      value: 'VerySatisfied',
+                    },
+                    {
+                      label: 'Satisfied',
+                      value: 'Satisfied',
+                    },
+                    {
+                      label: 'Slightly dissatisfied',
+                      value: 'SlightlyDissatisfied',
+                    },
+                    {
+                      label: 'Not satisfied at all',
+                      value: 'NotSatisfiedAtAll',
+                    },
+                  ]}
+                />
+                <FormFieldTextArea
+                  name="additionalFeedback"
+                  label="Additional feedback (optional)"
+                  maxLength={2000}
+                />
+                <ButtonGroup>
+                  <Button type="submit" disabled={formState.isSubmitting}>
+                    Send
+                  </Button>
+                </ButtonGroup>
+              </Form>
+            ) : (
+              <Panel headingTag="h2" title="Feedback received">
+                <p>Thank you.</p>
+              </Panel>
```

Comment:
One small a11y improvement might be to wrap this Panel with a `aria-live="polite"` to just notify screen readers when form disappears what's going on
```tsx
<div aria-live="polite">
    <Panel>...</Panel>
</div>
````